lib/experimental-default-theme.json

```diff
@@ -236,5 +236,8 @@
 				}
 			}
 		}
+	},
+	"styles": {
+		"spacing": { "blockGap": "30px" }
```

Comment:
@jasmussen what should be the default gap used? What do we do currently in CSS and where are the rules adding the margins?

Reply:
The default block margin as defined in _variables.scss has, for the longest time, been 28px. However that value was arrived upon based on a not-grid-based 2 * 14px "block padding" which we had in the pre-G2 block UI design, so it's entirely arbitrary. 

In classic themes, that margin is now stored in classic.scss:

```
html :where(.wp-block) {
  max-width: 840px;
  margin-top: 28px;
  margin-bottom: 28px;
}
```

(I'm debugging #32659 hence the different syntax)

In block themes at the moment we don't add any top/bottom margins to `.wp-block` at all. 

I would love for us to pick a base8 or base12 value, such as 24px, 32px or 36px. Here's what 24px looks like with a 16px font size and 32px line-height:

<img width="892" alt="Screenshot 2021-08-05 at 11 13 39" src="https://user-images.githubusercontent.com/1204802/128324995-ad665c88-524f-4b28-8d66-a3cba0ff2562.png">

That would also be nice as the default font size from the browser is 16px, which would mean that a 24px gap would equate to 1.5em. Nicely rounded numbers all around.

Reply:
Got it, I updated and used 24px.